src/main/java/com/chartboost/sdk/sample/ChartboostSample.java

```diff
@@ -135,42 +135,7 @@ public void onNothingSelected(AdapterView<?> parent) {
                     break;
 
                 case IN_PLAY:
-                    final CBInPlay inPlay = CBInPlay.getInPlay(location);
-                    if (inPlay == null) {
-                        addToUILog("In Play was not ready at " + location);
-                        break;
-                    }
-                    Bitmap inPlayBitmap = null;
-                    try {
-                        inPlayBitmap = inPlay.getAppIcon();
-                    } catch (Exception ex) {
-                        String exceptionAsString = Log.getStackTraceString(ex);
-
-                        addToUILog(exceptionAsString);
-                    }
-                    if (inPlayBitmap == null) {
-                        addToUILog("Unable to get InPlay bitmap at " + location);
-                        break;
-                    }
-                    inPlayIcon.setImageBitmap(inPlayBitmap);
-                    inPlayAd.setVisibility(View.VISIBLE);
-                    inPlay.show();
-                    addToUILog("In Play shown at " + location);
-                    inPlayShowing = true;
-
-                    inPlayIcon.setOnClickListener(v1 -> {
-                        if (inPlay != null) {
-                            inPlay.click();
-                            inPlayAd.setVisibility(View.GONE);
-                            addToUILog("In Play clicked at " + location);
-                            inPlayShowing = false;
-                        }
-                    });
-
-                    inPlayCloseButton.setOnClickListener(v12 -> {
-                        inPlayAd.setVisibility(View.GONE);
-                        inPlayShowing = false;
-                    });
+                    addToUILog("In Play is not supported");
```

Comment:
Nice. Why don't we just remove everything related to InPlay? They aren't working so...

Reply:
I guess I can remove the button